pipeline/alert.go

```diff
@@ -1263,6 +1263,9 @@ type AlertaHandler struct {
 	// List of Correlated
 	Correlate []string `tick:"Correlated" json:"correlate"`
 
+	// Map of alert Attributes
+	Attributes map[string]string `tick:"Attribute" json:"attributes"`
```

Comment:
In alerta attributes can be of different types (string, number, bool, etc), so it would be better if it was `map[string]interface{}`

Reply:
Nice point! I've completely missed this behaviour in Alerta. On my way with appropriate changes.